cmake/BuildLibCURL.cmake

```diff
@@ -149,6 +150,15 @@ macro(curl_build)
                 --disable-progress-meter
                 --disable-dnsshuffle
                 --disable-alt-svc
+                --disable-hsts
```

Comment:
HSTS is enabled in #7119 (and I don't see a reason to disable it if it does not engage extra dependencies).

Reply:
```diff
--- a/cmake/BuildLibCURL.cmake
+++ b/cmake/BuildLibCURL.cmake
@@ -150,7 +150,6 @@ macro(curl_build)
                 --disable-progress-meter
                 --disable-dnsshuffle
                 --disable-alt-svc
-                --disable-hsts
                 --enable-libcurl-option
                 --enable-openssl-auto-load-config
                 --enable-socketpair
```

Reply:
We agreed to list all options explicitly, but here you just removed it.

AFAIR, alt-svc and dnsshuffle also different between branches.

Reply:
> We agreed to list all options explicitly, but here you just removed it.

Fixed.

> AFAIR, alt-svc and dnsshuffle also different between branches.

Fixed.